### DISCUSSION TITLE
A deep dive into the Elixir AST: Building a static code analyzer

### DISCUSSION THREAD

**Discussion by giscus** [doorgan/blog#7] [General]
# A deep dive into the Elixir AST: Building a static code analyzer

Part 3 of The Elixir AST series. Analyzing the AST to build a static code analyzer.

https://dorgan.netlify.app/posts/2021/04/the_elixir_ast_analyzer/

**Comment by alichz2001** [comment]
very helpful. thanks.

**Reply by alaadahmed** [reply]
Thank you very much for the time and efforts to write these wonderful articles, they are very good reference for me and I bookmarked them. Really appreciate that for you.

**Comment by MatsLanGoH** [comment]
Thank you for this insightful and well-structured write-up about the Elixir AST! I am just getting started with Elixir, and this introduction was great to get a first view "under the hood" of the language.
